docs/src/pages/style/SvgIcons.js

```diff
@@ -1,49 +1,56 @@
 // @flow weak
 
 import React from 'react';
-import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import { withStyles } from 'material-ui/styles';
-import blue from 'material-ui/colors/blue';
 import green from 'material-ui/colors/green';
-import red from 'material-ui/colors/red';
 import SvgIcon from 'material-ui/SvgIcon';
 
-const styles = theme => ({
-  icon: {
-    margin: theme.spacing.unit,
-  },
-  iconBlue: {
-    fill: blue[500],
+const styles = {
+  root: {
+    display: 'flex',
+    alignItems: 'flex-end',
+    justifyContent: 'space-around',
+    width: '70%',
   },
   iconHover: {
-    fill: red[500],
     '&:hover': {
       fill: green[200],
     },
   },
-});
+};
+
+type ProvidedProps = {
```

Comment:
Please remove any reference to flow to the documentation.

Reply:
Most people copy & past our demos. We can't be fast forward toward  the ES specification.